cli/options.go

```diff
@@ -314,7 +314,7 @@ func ProjectFromOptions(options *ProjectOptions) (*types.Project, error) {
 		} else if nameFromEnv, ok := options.Environment[ComposeProjectName]; ok && nameFromEnv != "" {
 			opts.Name = nameFromEnv
 		} else {
-			opts.Name = regexp.MustCompile(`(?m)[a-z]+[-_a-z0-9]*[a-z0-9]+`).FindString(strings.ToLower(filepath.Base(absWorkingDir)))
+			opts.Name = regexp.MustCompile(`(?m)[a-z]+[-_a-z0-9]*[a-z0-9]*`).FindString(strings.ToLower(filepath.Base(absWorkingDir)))
```

Comment:
`[-_a-z0-9]*[a-z0-9]*̀  is redundant, could be just `[-_a-z0-9]*`

Reply:
Thats right! Thx!